### ISSUE TITLE
Hovering over various icons has a small dot on top.

### ISSUE THREAD

**Issue by KinetekEnergy** [sanoojes/Spicetify-Lucid#53]
### Do you have the latest versions of Spicetify, Spotify and Lucid?

- [X] I'm using the latest versions of Spicetify, Spotify and Lucid

### Is there already an issue for your problem?

- [X] I have checked older issues, open and closed

### Is your problem definitely caused by the theme?

- [X] I have checked the app behavior without the theme and it is different

### Environment & Computer Info

```Markdown
- Spotify version: 1.2.44.405.g81fd6352 (Windows 11 Pro, 64 bit)
- Installed from: Spotify Site
- Spicetify version: v2.37.1
- Color scheme: Dark
- Extensions: None
- Custom apps: None
- Lucid installation method: Marketplace
```

### Description

1. When hovering over some icons, a small dot appears on top
2. For example, hovering over the play/pause icon or the add to playlist icon show this
3. Other icons like the home icon or Marketplace icon have the dot on the bottom for some reason

**Comment by sanoojes** [comment]
Yes, a small dot appears on hover. It seems to be related to a tooltip, possibly Tippy. I will try to remove it.
